project_stock_product_set/views/project_task_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_task_form2" model="ir.ui.view">
+        <field name="name">project.task.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project_stock.view_task_form2" />
+        <field name="groups_id" eval="[(4, ref('stock.group_stock_user'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button
+                    class="btn-primary"
+                    type="action"
+                    name="%(act_open_wizard_product_set_add_from_task)d"
+                    icon="fa-cubes"
+                    attrs="{'invisible': [('use_stock_moves','=',False)]}"
+                    groups="base.group_user"
```

Comment:
Why this group?

Reply:
For similarity to https://github.com/OCA/sale-workflow/blob/13.0/sale_product_set/views/sale_order.xml#L15 was added but not necessary.